build/pipelines/release.yml

```diff
@@ -84,671 +98,113 @@ variables:
-    - task: PublishSymbols@2
-      displayName: 'Publish app symbols to MSDL'
-      inputs:
-        symbolsFolder: '$(Build.SourcesDirectory)/appxsym-temp'
-        searchPattern: '**/*.pdb'
-        SymbolsMaximumWaitTime: 30
-        SymbolServerType: 'TeamServices'
-        SymbolsProduct: 'Windows Terminal Application Binaries'
-        SymbolsVersion: '$(XES_APPXMANIFESTVERSION)'
-        # The ADO task does not support indexing of GitHub sources.
-        indexSources: false
-        detailedLog: true
-      # There is a bug which causes this task to fail if LIB includes an inaccessible path (even though it does not depend on it).
-      # To work around this issue, we just force LIB to be any dir that we know exists.
-      # Copied from https://github.com/microsoft/icu/blob/f869c214adc87415dfe751d81f42f1bca55dcf5f/build/azure-nuget.yml#L564-L583
-      env:
-        LIB: $(Build.SourcesDirectory)
-        ArtifactServices_Symbol_AccountName: microsoftpublicsymbols
-        ArtifactServices_Symbol_PAT: $(ADO_microsoftpublicsymbols_PAT)
-
-
-- ${{ if eq(parameters.buildTerminalVPack, true) }}:
-  - job: VPack
-    displayName: Create Windows vPack
-    dependsOn: BundleAndSign
-    steps:
-    - checkout: self
-      clean: true
-      fetchDepth: 1
-      submodules: true
-    - task: PkgESSetupBuild@12
-      displayName: Package ES - Setup Build
-    - task: DownloadBuildArtifacts@1
-      displayName: Download Build Artifacts
-      inputs:
-        artifactName: appxbundle-signed
-        extractTars: false
-    - task: PowerShell@2
-      displayName: Rename and stage packages for vpack
-      inputs:
-        targetType: inline
-        script: >-
-          # Rename to known/fixed name for Windows build system
-
-          Get-ChildItem Microsoft.WindowsTerminal_*.msixbundle | Rename-Item -NewName { 'Microsoft.WindowsTerminal_8wekyb3d8bbwe.msixbundle' }
-
-
-          # Create vpack directory and place item inside
 
-          mkdir WindowsTerminal.app
+stages:
+  - stage: Build
+    displayName: Build
+    dependsOn: []
+    jobs:
+      - template: ./templates-v2/job-build-project.yml
+        parameters:
+          pool:
+            name: SHINE-INT-L # Run the compilation on the large agent pool, rather than the default small one.
+            demands: ImageOverride -equals SHINE-VS17-Latest
+          branding: ${{ parameters.branding }}
+          buildTerminal: ${{ parameters.buildTerminal }}
+          buildConPTY: ${{ parameters.buildConPTY }}
+          buildWPF: ${{ parameters.buildWPF }}
+          pgoBuildMode: ${{ parameters.pgoBuildMode }}
+          buildConfigurations: ${{ parameters.buildConfigurations }}
+          buildPlatforms: ${{ parameters.buildPlatforms }}
+          generateSbom: ${{ parameters.generateSbom }}
+          codeSign: ${{ parameters.codeSign }}
+          beforeBuildSteps: # Right before we build, lay down the universal package and localizations
+            - task: PkgESSetupBuild@12

```

Comment:
This stuff is specific to the release build, and so it gets inserted by release.yml ONLY